include/crypto/types.h

```diff
@@ -29,4 +29,6 @@ typedef struct dsa_st DSA;
 typedef struct ecx_key_st ECX_KEY;
 # endif
 
+typedef struct symmetric_key_st SKEY;
```

Comment:
This is used in providers solely, so perhaps more clear naming would be:
```suggestion
typedef struct prov_skey_st PROV_SKEY;
```

Reply:
OK, agreed, thanks